services/project.go

```diff
@@ -0,0 +1,83 @@
+package services
+
+import (
+	. "clokify/models"
+	. "clokify/types"
+	"errors"
+	"log"
+)
+
+type ProjectServiceManager struct {
+	*ServiceManager
+}
+
+func (ps *ProjectServiceManager) CreateProject(project *Project, srv *ServiceManager) (error, bool) {
+	userService := &UserServiceManager{
+		ServiceManager: srv,
+	}
+
+	err, user := userService.GetUser(1)
+	if err != nil {
+		log.Fatal("user not found for given id to create project")
+	}
+	project.Users = append(project.Users, user)
+
+	projectErr, _ := ps.GetProjectByName(project.Name)
+	if projectErr == nil {
+		return errors.New("project already exists"), false
+	}
+
+	res := ps.Db.Model(&Project{}).Create(&project)
+	if res.Error != nil {
+		return errors.New("project not create successfully"), false
+	}
+	return nil, true
```

Comment:
return always after a line space it is good for code readability.

Reply:
yeah done